src/server/pfs/server/api_server.go

```diff
@@ -19,11 +18,6 @@ import (
 
 	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
-	"google.golang.org/grpc"
-)
-
-var (
-	grpcErrorf = grpc.Errorf // needed to get passed govet
```

Comment:
does `govet` work still (according to this comment)?

Reply:
Yeah, I forget exactly where we were using it, but it's unused now. It was only needed because for some reason `vet` didn't like us calling `grpc.Errorf` so we made this alias.